box-share-sdk/src/main/java/com/box/androidsdk/share/CollaborationUtils.java

```diff
@@ -76,15 +76,12 @@ public static String getCollaborationStatusText(Context context, BoxCollaboratio
         }
     }
 
-
-    public static void setInitialsThumb(Context context, TextView initialsView, int number) {
-        Drawable drawable = initialsView.getResources().getDrawable(R.drawable.initials_count_thumb_background);
-        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.ICE_CREAM_SANDWICH_MR1) {
-            initialsView.setBackground(drawable);
-        } else {
-            initialsView.setBackgroundDrawable(drawable);
+    public static String capitalizeFirstLetterOfEveryWord(String str) {
```

Comment:
We probably don't want this, we may need new strings to handle folder type and file type.